src/librustc_passes/rvalue_promotion.rs

```diff
@@ -248,7 +248,8 @@ impl<'a, 'tcx> CheckCrateVisitor<'a, 'tcx> {
         let tcx = self.tcx;
         let param_env = self.param_env;
         let region_scope_tree = self.tcx.region_scope_tree(item_def_id);
-        euv::ExprUseVisitor::new(self, tcx, param_env, &region_scope_tree, self.tables, None)
+        let tables = self.tables;
+        euv::ExprUseVisitor::new(self, tcx, param_env, &region_scope_tree, tables, None)
```

Comment:
hmm we may want to actually file an issue for this one tagged with NLL, in terms of tracking things that regress with respect to AST-borrowck.

Or we can just wait to someone to report it from the wild.  :)

Reply:
There's already #51915